core/src/test/java/org/infinispan/remoting/rpc/RpcManagerTest.java

```diff
@@ -0,0 +1,180 @@
+package org.infinispan.remoting.rpc;
+
+import static org.infinispan.remoting.responses.SuccessfulResponse.SUCCESSFUL_EMPTY_RESPONSE;
+import static org.testng.AssertJUnit.assertEquals;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.CompletionStage;
+
+import org.infinispan.commands.remote.ClusteredGetCommand;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.infinispan.remoting.responses.Response;
+import org.infinispan.remoting.responses.ValidResponse;
+import org.infinispan.remoting.transport.Address;
+import org.infinispan.remoting.transport.impl.MapResponseCollector;
+import org.infinispan.remoting.transport.impl.SingleResponseCollector;
+import org.infinispan.test.Exceptions;
+import org.infinispan.test.MultipleCacheManagersTest;
+import org.infinispan.test.TestingUtil;
+import org.testng.annotations.Test;
+
+/**
+ * @author Dan Berindei
+ * @since 9.2
+ */
+public class RpcManagerTest extends MultipleCacheManagersTest {
```

Comment:
we no longer annotate the class with the groups and test name?

Reply:
I was trying to see what happens if I remove all the test names at one point, maybe I was influenced by that. Or maybe I just forgot :)